### DISCUSSION TITLE
Why is Count Missing

### DISCUSSION THREAD

**Discussion by jrock2004** [prisma/prisma#23285] [Q&A] [answered]
### Question

So I am not sure why count and totalCount are missing from the type definition. 

![image](https://github.com/prisma/prisma/assets/655716/69301539-b641-44c4-8dfb-e04b79dae86e)

### How to reproduce (optional)

Clone repo https://github.com/jrock2004/customer-dashboard

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

model Customer {
  id          Int          @id @default(autoincrement())
  firstName   String
  lastName    String
  email       String       @unique
  phone       String
  imgUrl      String?
  initials    String
  createdAt   DateTime     @default(now())
  updatedAt   DateTime     @updatedAt
  memberships Membership[]
}

model Membership {
  id          Int      @id @default(autoincrement())
  title       String
  count       Int
  totalCount  Int
  description String
  price       String
  memberId    Int
  createdAt   DateTime @default(now())
  updatedAt   DateTime @updatedAt
  member      Customer @relation(fields: [memberId], references: [id])
}

```

```ts
import { PrismaClient } from "@prisma/client";

const prismaClientSingleton = () => {
  return new PrismaClient();
};

declare global {
  var prisma: undefined | ReturnType<typeof prismaClientSingleton>;
}

const prisma = globalThis.prisma ?? prismaClientSingleton();

export default prisma;

if (process.env.NODE_ENV !== "production") globalThis.prisma = prisma;

```

- OS: macOS
- Database: SQLite
- Node.js version: v20.11.1

- Run `prisma -v` to see your Prisma version and paste it

Environment variables loaded from .env
prisma                  : 5.10.2
@prisma/client          : 5.10.2
Computed binaryTarget   : darwin-arm64
Operating System        : darwin
Architecture            : arm64
Node.js                 : v20.11.1
Query Engine (Node-API) : libquery-engine 5a9203d0590c951969e85a7d07215503f4672eb9 (at node_modules/.pnpm/@prisma+engines@5.10.2/node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Schema Engine           : schema-engine-cli 5a9203d0590c951969e85a7d07215503f4672eb9 (at node_modules/.pnpm/@prisma+engines@5.10.2/node_modules/@prisma/engines/schema-engine-darwin-arm64)
Schema Wasm             : @prisma/prisma-schema-wasm 5.10.0-34.5a9203d0590c951969e85a7d07215503f4672eb9
Default Engines Hash    : 5a9203d0590c951969e85a7d07215503f4672eb9
Studio                  : 0.499.0

**Comment by ludralph** [comment]
Hi @jrock2004 👋 

Thank you for raising this question.

Did you run `prisma generate` after making any changes to your schema to update the generated client? This step is crucial for reflecting any schema updates in the Prisma client's type definitions.

**Reply by jrock2004** [reply]
@ludralph yes. If I run prisma studio I see the columns there just fine

**Reply by ludralph** [reply, marked as answer]
@jrock2004, can you restart your IDE or your Typescript server?

**Reply by jrock2004** [reply]
@ludralph thank you, that did it. I thought I tried that but I guess I did not

**Reply by ludralph** [reply]
You're welcome 👍
